Use new "create repo from template" GitHub feature (#100)
moj-analytical-services/platform_user_guidance: 05-deploying_shiny_app.Rmd

9.  [Add users to the app](#add-users).
10. [Access the app](#access-the-app).

### Use the app template{#use-template}

> To create a new repo for the app, use the R Shiny app template:

Reviewer: ```suggestion
To create a new repository based on the Shiny app template:
```

> 1. Go to the [rshiny-template repo](https://github.com/moj-analytical-services/rshiny-template).

Reviewer: ```suggestion
1. Go to the [rshiny-template](https://github.com/moj-analytical-services/rshiny-template) repository.
```

> 2. Click __Use this template__.

Reviewer: GDS guidance is to use select as opposed to click, as not all users may be using a mouse or trackpad.

```suggestion
2. Select __Use this template__.
```

3. Fill in the form:
    + Owner: `moj-analytical-services`

>     + Name: The name of your app, e.g., `my-app`

Reviewer: We should probably change this to avoid using e.g.

```suggestion
    + Name: The name of your app, for example, `my-app`
```
